test/NuGet.Core.Tests/NuGet.Packaging.Test/SigningTests/CertificateUtilityTests.cs

```diff
@@ -26,22 +28,23 @@ public CertificateUtilityTests(CertificatesFixture fixture)
         }
 
         [Theory]
-        [InlineData("SHA256WITHRSAENCRYPTION", Oids.Sha256WithRSAEncryption)]
-        [InlineData("SHA384WITHRSAENCRYPTION", Oids.Sha384WithRSAEncryption)]
-        [InlineData("SHA512WITHRSAENCRYPTION", Oids.Sha512WithRSAEncryption)]
-        public void IsSignatureAlgorithmSupported_WhenSupported_ReturnsTrue(string signatureAlgorithm, string expectedSignatureAlgorithmOid)
+        [InlineData(Common.HashAlgorithmName.SHA256, RSASignaturePaddingMode.Pkcs1, Oids.Sha256WithRSAEncryption)]
```

Comment:
`RSASignaturePaddingMode.Pkcs1` could be removed as a parameter and inlined within the test, since it never varies.